### ISSUE TITLE
Nenio respec does't work

### ISSUE THREAD

**Issue by Kraehe13** [BarleyFlour/RespecMod#194]
When i try to respec Nenio in Game Version 2.1.0w with the mod version v1.10.13 after i finished with her first level i can't control any character anymore, only molving around the camera opening the menu with ESC. 

It works with the other characters i tried it so far (Seelah and Wenduag)

**Comment by ePiirainen** [comment]
Adding to the same issue here, for me none of the character's level ups work anymore in a save with all characters respecced. If I press the level up button the UI dissapears and only ESC can be pressed anymore, softlocking the game.

**Comment by maloadjav** [comment]
of course . its pathfinder s update

**Comment by Kraehe13** [comment]
> of course . its pathfinder s update

I know. I reported it because it was only with Nenio then. I could respec Wenduag and Seelah without problems. Yesterday i tried to respec Arue but had the same bug as with Nenio.

**Comment by ePiirainen** [comment]
> of course . its pathfinder s update

Just reporting an issue with "pathfinder update broke mod pls fix" doesn't really help whoever is updating the mod.

**Comment by Kraehe13** [comment]
> > of course . its pathfinder s update
> 
> Just reporting an issue with "pathfinder update broke mod pls fix" doesn't really help whoever is updating the mod.

I postet any detail i can post about it. I'm not a programmer. I postet when it happens and how it happens for me, dunno what additional information i could add.

**Comment by ePiirainen** [comment]
> > > of course . its pathfinder s update
> > 
> > 
> > Just reporting an issue with "pathfinder update broke mod pls fix" doesn't really help whoever is updating the mod.
> 
> I postet any detail i can post about it. I'm not a programmer. I postet when it happens and how it happens for me, dunno what additional information i could add.

Yeah I know, I replied to the other guy adding info everyone already knows

**Comment by Kraehe13** [comment]
Quick save (F5) and quick load (F8) resolves the problem.
